Fix static build

fbgrab links against libpng, which depends on libm. For shared library
builds, there is nothing special to do about this, but for static
library builds, it is necessary to pass -lm when linking fbgrab.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@
 all: fbgrab fbgrab.1.gz
 
 fbgrab: fbgrab.c
-	$(CC) -g -Wall $(CFLAGS) $(LDFLAGS) $< -lpng -lz -o $@
+	$(CC) -g -Wall $(CFLAGS) $(LDFLAGS) $< -lpng -lz -lm -o $@
 
 fbgrab.1.gz: fbgrab.1.man
 	$(GZIP) $(GZIPFLAGS) $< > $@
